src/scss/components/dropdown-select/index.scss

```diff
@@ -70,74 +66,74 @@ hx-select-control {
     border: none;
     padding-left: 0.75rem;
     width: 100%; // prevent overflow for lengthy option text
-
     // Adjust native trigger for MS browsers (mainly IE)
     // Edge will obscure it with <hx-select>.
-    #MS.set.expand({
+
+    &::-ms-set-expand {
```

Comment:
Incorrect pseudo-element. Please change to `&::-ms-expand`

Reply:
Changes have been done, please review.